clustered/server/src/main/java/org/ehcache/clustered/server/EhcachePassiveEntity.java

```diff
@@ -143,6 +144,20 @@ private void invokeLifeCycleOperation(LifecycleMessage message) throws ClusterEx
     }
   }
 
+  private void configure(ConfigureStoreManager message) throws ClusterException {
+    ehcacheStateService.configure(message);
+    ehcacheStateService.getClientMessageTracker().setEntityConfiguredStamp(message.getClientId(), message.getId());
+  }
+
+  private void trackAndApplyMessage(LifecycleMessage message) {
+    ClientMessageTracker clientMessageTracker = ehcacheStateService.getClientMessageTracker();
+    if (!clientMessageTracker.isAdded(message.getClientId())) {
```

Comment:
Except this one as this will change in going forward for #1211. See https://github.com/AbfrmBlr/ehcache3/blob/issue-1208-bckup/clustered/server/src/main/java/org/ehcache/clustered/server/EhcachePassiveEntity.java#L179